source/Components/List.cpp

```diff
@@ -43,16 +39,15 @@ namespace FileShare::GUI::Components {
         item->setWidth("100%");
         this->onSizeChangeSignals.push_back(
             item->onSizeChange.connect([this, widgetIndex]() {
-                this->updateItemsPosition(widgetIndex);
-                this->updateHeight();
-            })
+            this->updateItemsPosition(widgetIndex);
+            this->updateHeight();
+        })
         );
```

Comment:
Was good before :sweat_smile: 
```suggestion
                this->updateItemsPosition(widgetIndex);
                this->updateHeight();
            })
        );
```